### PR TITLE
(feat) Hook map bounds to API plurb requests

### LINKED ISSUE #98: (feat) Set+send mapBounds to backend

### PR DESCRIPTION
**Pull request by NMVW**
close #98

### CODE REVIEW

File: client/app/components/Map/Map.jsx [hunk 1]
```diff
@@ -1,22 +1,28 @@
 import React from "react";
-import store from '../../STORE.jsx';
-import ReactDOM from 'react-dom';
+import { store } from '../../STORE.jsx';
```
Comment (mkeith121) [inline client/app/components/Map/Map.jsx]: Shouldn't need store here.

Comment (NMVW) [inline client/app/components/Map/Map.jsx]: we have to pass store implicitly through the parent app component if we do not import it here. a future PR will make this change for all child components of the app if we so choose.

Comment (mkeith121) [inline client/app/components/Map/Map.jsx]: Check out index.jsx in the root directory

Comment (NMVW) [inline client/app/components/Map/Map.jsx]: Will this component dispatching cause the entire app to re-render?

**Pull request merged**